stl/inc/cstddef

```diff
@@ -24,7 +24,7 @@ using _CSTD ptrdiff_t;
 using _CSTD size_t;
 using max_align_t = double; // most aligned type
 
-#if _HAS_STD_BYTE
+#if __cpp_lib_byte

```

Comment:
These should all be `#ifdef` instead of `#if`, as detailed in #564.

Reply:
Yep you right !, thanks:)